docs/pages/docs/getting-started.en-US.mdx

```diff
@@ -7,7 +7,37 @@ import { Callout, Tabs, Tab } from 'nextra-theme-docs'
 
 # Getting Started
 
-## Installation
+## Quick setup
+
+It is recommended to set up your wagmi app using the `create-wagmi` command line interface (CLI). This will set up a new wagmi app using TypeScript and install the required dependencies:
+
+{/* prettier-ignore-start */}
+<Tabs items={['npm', 'pnpm', 'yarn']}>
+  <Tab>
+```bash
+npm init wagmi
+```
+  </Tab>
+  <Tab>
+```bash
+pnpm create wagmi
+```
+  </Tab>
+  <Tab>
+```bash
+yarn create wagmi
+```
+  </Tab>
+</Tabs>
+{/* prettier-ignore-end */}
+
+When the setup is complete, you can start your app by running `npm run dev` and then navigating to `http://localhost:3000`.
```

Comment:
Would be nice if we could make this package manager aware (based on the above tab selection).